Sources/Amplitude/Configuration.swift

```diff
@@ -32,10 +32,88 @@ public class Configuration {
     public var minTimeBetweenSessionsMillis: Int
     public var identifyBatchIntervalMillis: Int
     public internal(set) var migrateLegacyData: Bool
-    public var defaultTracking: DefaultTrackingOptions
+    public var autocapture: AutocaptureOptions
```

Comment:
consider making this a private(set), which will provide immutability but will still allow defaultTracking changes to be reflected internally.